docs/guide.md

```diff
@@ -0,0 +1,210 @@
+
+## Hello, PatternPack!
+Once you're done configuring PatternPack, you're ready to start it up for the first time. Open a command prompt and you have two options:
+
+```bash
+$ grunt patternpack:run
+# Or if you set up a default grunt task that points at patternpack:run
+$ grunt
+```
+
+This will:
+- Generate the files and folders you'll need
+- Build your Pattern Library in a temporary directory, `./html` or whatever you configured in your [`options.build`](https://github.com/patternpack/patternpack#build).
+- Start up a web server at `http://localhost:8888/`
+- Start up a watch task to auto build any changes
+
+## Adding a Pattern
+*Note that due to limitations in `grunt-watch`, you will need to restart PatternPack every time you add a new file for the watch to recognize it. Once it sees it, all changes with automatically reload.*
+
+Patterns are composed of two files:
+
+- A Markdown (`*.md`) file that contains all the content for a documentation page
+- A Sass/LESS file with the styles for that pattern
+
+To get started, add those two files into one of your Structure folders (if you kept the defaults, it'll be `./src/atoms`, `./src/molecules`, and `./src/templates`).
+
+You can put anything you'd like for the documentation page in the Markdown file, but a good starting point is this template:
+
+```md
+---
+title: Buttons
+---
+
+# Buttons
+Insert your documentation here
+
+## Example
+<div class="library__example">
+  Insert your live example here
+</div>
+
+## Code
+
+Insert your code example here
+TODO: INSERT BACKTICKS - HOW DO I ESCAPE THEM??
+```
+
+The `<div class="library__example">` strips out the default PatternPack styling from your example.
+
+The matching Sass/LESS file(s) will be globbed into a file called `_patternpack-patterns.scss or .less`. By default when you generate PatternPack for the first time, you will also get a `patterns.scss or .less` file (or what you configure in [`options.css.fileName`](https://github.com/patternpack/patternpack#cssfilename)) that will `@import` the `_patternpack-patterns` file.
+
+## Releasing Your Code
+PatternPack's most powerful feature is the ability to easily version and release your code, which allows other applications to pull in your styles and lock them to a specific version, preventing unexpected changes.
+
+*Note: The versioning and release process is built on the assumption you are using Git as your source control system. At this time, SVN, Mercurial, TFS (without Git), and others are not supported. However, because the assumption is that PatternPack will live in its own repository, you can set up a separate Git-based project without interrupting your app's development workflow.*
+
+In the default `grunt` configuration, you have a `patternpack:run` task. This task does a few things:
+
+1. Generates a new build in the `./dist` directory (or whatever you configured in [`options.release`](https://github.com/patternpack/patternpack#release)).
```

Comment:
I think it rebuilds your pattern library as the 1st step.  Then copies the files to the `./dist` directory